### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by samtstern**
This README gives more in-depth instructions on how to automate all the deployment steps here for Google Cloud and Firebase.  Hopefully it will make it a little easier for someone to get started!

### CODE REVIEW

**Comment by samtstern** [comment]
Sorry I did not mean to open this yet!

**Comment by achrinza** [comment]
Seems like this would make the README rather long-winded. Maybe we should shift the setup guides into their own page?

This would be my suggestion:

```
|
L README.md
L docs
    L setup-with-gcp.md
```

Within the README, we could have the following to direct users to the respective docs:

```md
## Setup

Setup guides are available for these platforms:
- [Google Cloud Platform / Firebase](./docs/setup-with-gcp.md)
```

This would help achieve 2 things:

1. Keep the README short and sweet
    A README is meant for users to quickly glance at what the repo is about, and provide links to resources for further reading,

2. Keep the setup guides compartmentalized
    We want to make the docs as easy to read as possible for the users. By splitting the guides according to their funtion, we can reduce the overall size of the docs and make it less daunting for new users to understand.

File: README.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ The OpenTrace reference implementation comprises:
...
 
 ## Prerequisites:
-+ [Node.js](https://nodejs.org/)
++ [Node.js](https://nodejs.org/) v10
 
 
 ## Create Firebase Project
 1. Create a new Firebase Project from [Firebase console](https://console.firebase.google.com/).
 2. Enable Google Analytics for the project, to be used for Firebase Crashlytics and Firebase Remote Config.
+3. Make sure to upgrade the project from the "Spark" free plan to the "Blaze" pay-as-you-go plan to avoid future quota issues.
```
Comment (achrinza) [inline README.md:29]: Not too sure if this is necessary as the responsibility for managing quota is on the user, regardless of what SaaS offering they're using.

**Comment by samtstern** [comment]
@achrinza keeping the README short seems reasonable! I am happy to move the setup guide to a `docs` subfolder.

Also since I am new here: are you on the project team or just passing by for a review?

**Comment by achrinza** [comment]
@samtstern I’m watching the repo and passing by for a review, though I’m not on the project team.

**Review by qtangs — APPROVED** [review]

**Pull request merged**